### PR TITLE
fix: Corrected avatar positions due to new cockpit seats

### LINKED ISSUE #8356: FIRSTOFFICER_AVATAR is hovering over the seat (if activated)
### Aircraft Version

Development

### Build info

```json
{
    "built": "2023-12-29T21:01:10+00:00",
    "ref": "refs/heads/master",
    "sha": "68bf94051aa8f37925b5b629624c80c9813721be",
    "actor": "beheh",
    "event_name": "manual",
    "pretty_release_name": "master:68bf9405",
    "version": "v0.12.0-dev.68bf940"
}
```

### Describe the bug

The placement of the FIRSTOFFICER_AVATAR looks rather punishing and stressfull as the backside is hovering a couple of cm above the seat and distant to the backrest, too.

### Expected behavior

The placement of the avatars should be a bit closer to reality. At least seat and backrest should be touched.

### Steps to reproduce

1. Activate the F/O in the EFB settings
2. Look to your right
3. Probably correct your own position a bit backwards
4. Voilá, you'll see it!

### References (optional)

The screenshot is brightened to make it clearer:
2023-12-22 20_16_44-FSab

### Additional info (optional)

Had been an issue in all versions, but could be corrected in "**A320_NEO_INTERIOR_LOD00.gltf**"
```
ORIGINAL: ,{"translation":[-0.5676980018615723,1.3970900774002076,10.977500915527344],"name":"FIRSTOFFICER_AVATAR"}
CORRECTED: ,{"translation":[-0.568,1.327,10.937],"name":"FIRSTOFFICER_AVATAR"}  -> difference (left,up,fwd): 0,-0.7,-0.04 and rounded afterwards.
```
This is still a compromise as the seat's inclination doesn't fit exactly the avatar's design, however, it looks better, but is a first attempt (the correction might be a bit overdone).  

BTW, as an engineer I think the values (the unit is Meters!) should be rounded to max. 3 decimal places anyway    :-)

### Discord Username (optional)

_No response_

### PR DESCRIPTION
**Pull request by frankkopp**
Fixes #8356

## Summary of Changes
Corrected avatar positions due to new cockpit seats

### Note
A bit unclear which file to change as there is no documentation.
There are two A320_NEO_INTERIOR_LOD00.gltf - one in base and one in src/model and there is also a model.json with the avatar nodes. Only changing the base gltf worked - so either this is wrong and it should be done in the others or the other copies of the avatar node incl. the models.json are obsolete. 

Discord username (if different from GitHub): cdr_maverick

## Screenshots
image

## Testing instructions
Check the position of the avatars (turn them on in the flyPad settings).
Check possible other side effects/regression. 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by Flaxx-Rat** [comment]
The result looks okay for me. Unfortunately I'm not able following you check procedures.

**Comment by tracernz** [comment]
The model is exported from blender now, so we don't hand edit the gltf.

**Comment by frankkopp** [comment]
> The model is exported from blender now, so we don't hand edit the gltf. The plan is to do another model update with an accumulation of fixes soon (doing them in small PRs is too expensive while we don't have LFS).

Can we clean up some already or at least document what is still required and not?

**Review by 2hwk — APPROVED** [review]

**Comment by tracernz** [comment]
So, these changes are already made in blender (thanks to @Watsi01 for the positions). They will become part of the model next time we do an export (along with a bunch of other changes queued up). I think this can close?

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR_LOD00.gltf
Comment (tracernz) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR_LOD00.gltf:1]: This file should not be hand-edited. The changes need to be made in the blend file, exported, and built in MSFS dev mode, just as for the A380.

Comment (frankkopp) [reply fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR_LOD00.gltf:1]: Well, how about documenting stuff then?
Adding a README.md is a NOT lot of work but saves time for others. I really don't understand why people have so little respect for others and expect others too read minds.

**Comment by Flaxx-Rat** [comment]
> So, these changes are already made in blender (thanks to @Watsi01 for the positions). They will become part of the model next time we do an export (along with a bunch of other changes queued up). I think this can close?

Half a year later I can't see any change (I'm using the DEV-version only). My F/O still hovers over the seat and therefore **I would prefer re-opening this issue** as it obviously had been my misunderstanding that this fix would be introduced soon.

However, thanks for your efforts guys. I love what you are doing.

**Comment by Maximilian-Reuter** [comment]
> > So, these changes are already made in blender (thanks to @Watsi01 for the positions). They will become part of the model next time we do an export (along with a bunch of other changes queued up). I think this can close?
> 
> Half a year later I can't see any change (I'm using the DEV-version only). My F/O still hovers over the seat and therefore **I would prefer re-opening this issue** as it obviously had been my misunderstanding that this fix would be introduced soon.
> 
> However, thanks for your efforts guys. I love what you are doing.

This is a PR not an issue. The issue #8356 you closed yourself. If you want to reopen it, go ahead. It is tracked in the model fixes issue though sot really necessary.

It will be fixed when we generate a new model update. No ETA though.

**Comment by Flaxx-Rat** [comment]
Thanks for clarifying. You're right that I'd closed it myself. The reason was my wrong assumption about the state. My understanding now is that it's in the pipe and thus fully sufficient for me (nothing really breaking anything important for flying the bird). If it's not about to get lost - fine!

**Pull request closed without merging**